pkgs/servers/web-apps/ackee/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, mkYarnPackage
+, nodejs
+, python2
```

Comment:
Oh no! I just noticed this! 🚨 `python2` is dead and we shouldn't be adding any new usage of it to nixpkgs! What about `python3`?